bindings/java/com/google/iree/native/instance_wrapper.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "bindings/java/com/google/iree/native/instance_wrapper.h"
 
+#include <mutex>
+
```

Comment:
Coming from main -> google, do we need this include?

Reply:
What do you mean?

https://en.cppreference.com/w/cpp/header/mutex
```
  static std::once_flag setup_vm_once;
  std::call_once(setup_vm_once, [] { SetupVm(); });
```

Looks like Bazel was fine without the include, but CMake/clang needed it.

Reply:
Oh I see, thanks! Just feeling weird that there are no any other changes in the cc file.